scripts/add_url.py

```diff
@@ -3,16 +3,11 @@
 
 from pymongo import MongoClient
 
-# add root directory to python path
-sys.path.append(os.path.abspath(os.path.dirname(__file__) + '/../'))
 
-from config.config import Config
-from dao import Dao
-
-DATABASE_NAME = config.get_db_name()
+DATABASE_NAME = 'garpr'
 TOURNAMENTS_COLLECTION_NAME = 'tournaments'
 PENDING_TOURNAMENTS_COLLECTION_NAME = 'pending_tournaments'
-mongo_client = MongoClient(host=config.get_mongo_url())
+mongo_client = MongoClient(host='mongodb://devuser:devpass01@127.0.0.1/admin')
```

Comment:
hmm we really should get the parameters from config, since they are different on stage/prod.

if the `sys.path.append(os.path.abspath(os.path.dirname(__file__) + '/../'))` isn't working for some reason then it is okay if the script has to be run from the root directory.